examples_tests/common/CommonAPI.h

```diff
@@ -163,8 +164,9 @@ class CommonAPI
 
 									auto oldEvent = *(channelEvents.end() - rewindBack);
 
-									if(oldEvent.timeStamp < minEventTimeStamp) {
-										minEventTimeStamp = oldEvent.timeStamp;
+									// Which oldEvent of channels are earlier?
```

Comment:
earlier means lower timestamp, I think you're looking for the maximal timestamp so you're looking for "most recent" not "earlier", I think you have a comment to correct